gensim/models/phrases.py

```diff
@@ -592,23 +649,26 @@ def pseudocorpus(self, phrases_model):
                             phrases_model.common_terms)
 
     def score_item(self, worda, wordb, components, scorer):
-        """score is retained from original dataset
-        """
+        """Score, retained from original dataset."""
         try:
             return self.phrasegrams[tuple(components)][1]
         except KeyError:
             return -1
 
     def __getitem__(self, sentence):
-        """
-        Convert the input tokens `sentence` (=list of unicode strings) into phrase
+        """Convert the input tokens `sentence` (=list of unicode strings) into phrase
```

Comment:
don't use this `(=list of unicode strings)`, better to write concrete types for arguments.